src/GlobalErrorBoundary.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as branding from './branding'
+import { msg } from './intl'
+import './error_styles-nomodules.css'
+import AppConfiguration from './config'
+
+class GlobalErrorBoundary extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = { hasError: false }
+    this.props.errorBridge.setErrorHandler((error) => {
```

Comment:
That is nice and clean!

Only suggestion is to add a blank line and a comment above just explaining that the component is registering itself as a generic error handler.

I know the component is only used once in the App so it isn't necessary, but in any other component, you'd want to register in `componentDidMount` and unregister `componentWillUnmount` to avoid any memory leaks.  Otherwise the error bridge could hang on to every instance of the component (and its children).  That would be bad.